ml-agents/mlagents/trainers/tests/torch/test_networks.py

```diff
@@ -78,7 +78,7 @@ def test_networkbody_visual():
     sample_obs = torch.ones((1, 84, 84, 3))
     sample_vec_obs = torch.ones((1, vec_obs_size))
 
-    for _ in range(150):
+    for _ in range(1000):
```

Comment:
Is this intended? It could slow down this test by a bit.

Reply:
No, good catch!